omnibus/package-scripts/chef-server/postinst

```diff
@@ -11,6 +11,45 @@ function error_exit
 	exit 1
 }
 
+version=$(awk '/chef-server/{ split($2,a,"+");print a[1];exit }' /opt/opscode/version-manifest.txt)
```

Comment:
Try to see if we can decide if a migration is needed based on info in var/opt/opscode/upgrades/migration-level from the api.

Reply:
Deciding to not use the logic in preinst and postinst script since, the migration api is not available to us before the install.